webapp/channels/src/components/admin_console/license_settings/modals/confirm_license_removal_modal.tsx

```diff
@@ -58,8 +58,8 @@ const ConfirmLicenseRemovalModal: React.FC<Props> = (props: Props): JSX.Element
                 <div className='content-body'>
                     <div className='no-header__img alert-svg'>
                         <AlertSvg
-                            width={150}
-                            height={150}
+                            width={130.5}
+                            height={130.5}
```

Comment:
Update height to:
```
height={103.5}
```

And the `margin-top: -14px` on `alert-svg` should be removed and add `margin-bottom: 15px` instead

Reply:
Fixed